extensions/Push/lib/Connector/Phabricator.pm

```diff
@@ -59,9 +54,32 @@ sub should_send {
 }
 
 sub send {
+    my ( $self, $message ) = @_;
+
     my $logger = Bugzilla->push_ext->logger;
 
-    $logger->info('AUDIT');
+    my $data = $message->payload_decoded;
+    my $bug_data = $self->_get_bug_data($data) || return 0;
+    my $bug = Bugzilla::Bug->new( { id => $bug_data->{id}, cache => 1 } );
+
+    if ( any { $_->{name} =~ /core-security/ } @{$bug->groups()} ) {
+        $logger->info('HAS SECURITY!');
+
+        my @attachments = grep {
+            $_->isobsolete == 0 &&
+            $_->contenttype eq PHAB_CONTENT_TYPE &&
+            $_->{attacher}->{realname} eq 'phabricatorbot'
+        } @{ $bug->attachments() };
+
+        if(@attachments){
+            $logger->info('HAS PHAB ATTACHMENTS!');
+
+            my @attachment_ids = map { grep(PHAB_ATTACHMENT_PATTERN, $_->filename) } @attachments;
```

Comment:
I don't think this will work the way we want it to in that I don't think grep will return the value inside () but instead the full string of $_.

So we should do this instead:

my @rev_ids;
foreach my $attachment (@attachments) {
      my ($rev_id) = ($attachment->filename =~ PHAB_ATTACHMENT_PATTERN);
      next if !$rev_id;
      push(@rev_ids, $rev_id);
}